packages/remix-react/transition.ts

```diff
@@ -221,6 +221,12 @@ type FetcherStates<TData = any> = {
     submission: ActionSubmission;
     data: TData;
   };
+  LoadingSubmissionActionRedirect: {
```

Comment:
Let's rename this type to `LoadingActionRedirect`

Reply:
👍 [27e6b23](https://github.com/remix-run/remix/pull/2188/commits/27e6b23fddd3cccd68bb2066b0c412ac75efd3f1)